roles/openshift_descheduler/defaults/main.yaml

```diff
@@ -1,7 +1,6 @@
 ---
 # descheduler common setup
 openshift_descheduler_state: present
-openshift_descheduler_namespace: openshift-infra
```

Comment:
imho s/openshift-infra/openshift-descheduler would make the PR smaller and easier to change the namespace in the future

Reply:
As discussed offline, we do not want to make it configurable.

Reply:
What a shame the Ansible has no concept of constant variables then.